app/account/services_views.py

```diff
@@ -0,0 +1,24 @@
+import typing as tp
+
+
+def create_new_account(data: tp.Dict[str, tp.Union[str, bool, int]]) -> tp.Dict[str, tp.Union[str, bool, int]]:
+    """
+    Create new account
+    Args:
+        data: register data
+    Return:
+        dict with account data
+    """
+
+    from main import db
```

Comment:
local imports again

Reply:
I need help with this because if I passed an external function import, I will import this function to views and then to main when I register all urls, or I can do it after another function

Reply:
done